oc/openshift_ops.py

```diff
@@ -184,3 +185,165 @@ def create_project(self, project):
             raise Exception(err)
 
         return _rc
+
+    def create_cephblockpool(
+        self,
+        namespace,
+        cephblockpool_name_prefix="autotests-cephblockpool",
+        **kwargs
+    ):
+        """
+        Creates cephblock pool
+
+        Args:
+            namespace (str): namespace to create cephblockpool
+            cephblockpool_name_prefix (str): prefix given to cephblockpool
+
+        kwargs:
+            The keys, values in kwargs are:
+                - failureDomain : (str)
+                - size : (int)
+
+        Returns:
+            str : name of the cephblockpool created
+
+        Raises:
+            KeyError when error occured
+
+        Examples:
+            create_cephblockpool(
+                namespace,
+                cephblockpool_name_prefix="autotests-blockpool",
+                failureDomain="host",
+                size=3)
+
+        """
+        service_cbp = self.dyn_client.resources.get(
+            api_version='v1',
```

Comment:
Let's maintain the api_version in defaults so we will be able to change a single place in the future

Reply:
Done